numpy/core/src/multiarray/temp_elide.c

```diff
@@ -82,7 +82,12 @@
 #define NPY_MIN_ELIDE_BYTES (32)
 #endif
 #include <dlfcn.h>
+
+#if defined HAVE_EXECINFO_H
 #include <execinfo.h>
+#elif defined HAVE_LIBUNWIND_H
+#include <libunwind.h>
+#endif
```

Comment:
Thanks, this makes sense to me now, although if neither is found, I guess the whole thing should be disabled (which is #61)

Reply:
Yes, `HAVE_BACKTRACE` should disabled those include if `backtrack()` not found. Unless new backtracking library showed up, which should throw an error to notice devs.